packages/suite/src/support/suite/ErrorBoundary.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import { H1, P, Button } from '@trezor/components';
+import { SENTRY_URL } from '@suite/config/suite/sentry';
+import {
+    HEADER_HEIGHT,
+    HEADER_HEIGHT_UNIT,
+    FOOTER_HEIGHT,
+    FOOTER_HEIGHT_UNIT,
+} from '@suite/constants/suite/layout';
+
+import * as Sentry from '@sentry/browser';
+
+console.log(SENTRY_URL);
+
+Sentry.init({ dsn: SENTRY_URL });
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-height: calc(
+        100vh -
+            (
+                ${`${HEADER_HEIGHT}${HEADER_HEIGHT_UNIT}`} +
```

Comment:
why separate var for unit?

Reply:
it would be way better if we could figure out how to doit without that ugly min-height prop (maybe place error boundery inside the component where it is specified? or just remove it and let errorboundary take whole viewport?)

Reply:
removed